docs/Q-and-A.md

```diff
@@ -512,10 +516,49 @@ _(@henkvancann)_
 However there are also votes for `did:keri`: _Drummond Reed_ (Dec 2 2020): "at IIW we asked that question and  feedback overwhelmingly favored did:keri. Furthermore, I’ve proposed that the keri namespace be reserved within the method-specific ID spaces of other DID methods as well, The Indy community has agreed to reserve the keri namespace in the Indy DID method."\
 _(@henkvancann)_
 
+## Some say that with KERI, a DID can be reduced to did:<identifier>. But that’s not a valid DID?!
+_Every DID must have a method name component before the method-specific ID._
+
+The first paper mentioning the absence of the method is [Thinking of DID? KERI On](https://humancolossus.foundation/blog/thinking-of-did-keri-on) by The Human Colossus Foundation, written by Robert Mitwicki. He addressed the concern in the question (the invalidity of the DID method) and made it more clear what the Foundation meant by that: "We look into the future and with that view we think that namespace could be dropped and we could keep only identifier, as the namespace seems to be one of the major drawbacks of decentralized identifiers at the moment. In my opinion `did:keri:<identifiers>`  would be just intermediate step as the issue addressed by post would still hold. We see that KERI could be a major upgraded for DID; not replacement."
+
 ## Is it possible to create a KERI DID that is permanently locked to the "did:key" style / ephemeral?
 It is. Sections 2.2.3 - 2.3.1 of the [white paper](https://github.com/SmithSamuelM/Papers/blob/master/whitepapers/KERI_WP_2.x.web.pdf) explains transferrability and the "basic"-type identifier, which always represents a public key and may be either transferrable (can be updated) or non-transferrable (ephemeral/did:key style). section 14.2 actually details how these are encoded, basically check the first few chars of the identifier.\
 _(CharlesCunningham)_
 
+## Why do you re-invent Ripple and claim it's something new?
+To begin with KERI has no blockchain, and doesn't depend on blockchains. If an implementation depends on blockchains at all, KERI operates blockchain agnostic.
+Secondly KERI doesn't support a crypto currency. It doesn't need because it can easily connect to one if needed and again being crypto currency agnostic while doing so.
+Lastly KERI is fundamentally different from Ripple: it doesn't need total ordering, timestamping and Proof of Authority consensus on transactions registered on a ledger.
+
+It's comparing apples and oranges. But we're happy to do that exercise for the hard-to-convince part of the SSI community.
+{TBW prio 1}
+#### Differences between Ripple consensus and KERI witness ambient availability of KELRs
+{TBW prio 2}
```

Comment:
Might be useful to link directly to Mazière's work here:
https://www.stellar.org/developers/guides/concepts/scp.html

Reply:
I think that is done, by taking another generalist angle to it.

## **Q: Why do you reinvent blockchains and claim it's something new?
To begin with KERI has no blockchain, and doesn't depend on blockchains. If an implementation of KERI depends on blockchains at all, KERI operates blockchain agnostic.
Secondly KERI doesn't support a crypto currency. It doesn't need currency because it can easily connect to one, if needed. And again, KERI is crypto currency agnostic while doing so.
Lastly KERI is fundamentally different from blockchains like Ripple (Permissioned PBFT consensus) or Stellar (imcomplete open public, non-permissioned PBFT consensus): it doesn't need **total ordering**, timestamping and Proof of Authority consensus on transactions registered on a ledger.

It's comparing apples and oranges. But we're happy to do that exercise for the hard-to-convince part of the SSI community.